tests/validators.py

```diff
@@ -496,8 +496,8 @@ def validate_cis_hardening():
     print(output)
     assert "41 checks WARN" in output
     if os.environ.get("STRICT") == "yes":
-        assert "82 checks PASS" in output
-        assert "1 checks FAIL" in output
```

Comment:
This does not seem right. I though that in the strict branch we are failing one test. Maybe the strict branch has expire and it falls back to classic where all test are passing.